api/leetcode.js

```diff
@@ -1,54 +1,58 @@
-import { dailyGetLeetcodeData, weeklyGetLeetcodeData, URL } from './leetcode/index.js'; 
-import { config } from '../config.js';
-import { MessageEmbed } from 'discord.js';
+import { MessageEmbed } from "discord.js";
+import { dailyGetLeetcodeData, weeklyGetLeetcodeData, URL } from "./leetcode/index.js";
+import { config } from "../config.js";
 
 export class Leetcode {
-    client
-    channel
+    client;
+
+    channel;
 
     constructor(discordClient) {
-        this.client = discordClient
+        this.client = discordClient;
     }
-    
+
     _getLeetCodeChannel = async () => {
-        let dominationGuild = this.client.guilds.resolve(config.GUILD_ID);
+        const dominationGuild = this.client.guilds.resolve(config.GUILD_ID);
         return dominationGuild.channels.cache.find(
-            (channel) => channel.name === config.LEETCODE_CHALLENGES_CHANNEL
+            (channel) => channel.name === config.LEETCODE_CHALLENGES_CHANNEL,
         );
-    }
+    };
 
-    _postDailyIntoChannel = async ({frontendQuestionId, title, link , difficulty, acRate}) => {
+    _postDailyIntoChannel = async ({
+        frontendQuestionId, title, link, difficulty, acRate,
+    }) => {
         const message = new MessageEmbed()
-            .setColor('#00FFFF')
+            .setColor("#00FFFF")
             .setTitle(`${frontendQuestionId}. ${title}`)
             .setURL(`${URL}${link}`)
             .addFields(
-                { name: 'Difficulty', value: "```" + difficulty + "\n```", inline: true },
-                { name: 'Success rate', value: "```" + Number.parseFloat(acRate).toFixed(2) + "```", inline: true },
+                { name: "Difficulty", value: `\`\`\`${difficulty}\n\`\`\``, inline: true },
```

Comment:
this one looks weird:
\`\`\`${difficulty}\n\`\`\``

Reply:
thanks, fixed.